vhdbuilder/packer/install-dependencies.sh

```diff
@@ -471,14 +471,32 @@ for AZUREDISK_CSI_VERSION in ${AZUREDISK_CSI_VERSIONS}; do
 done
 
 AZUREFILE_CSI_VERSIONS="
-0.3.0
+0.7.0
 "
 for AZUREFILE_CSI_VERSION in ${AZUREFILE_CSI_VERSIONS}; do
   CONTAINER_IMAGE="mcr.microsoft.com/k8s/csi/azurefile-csi:v${AZUREFILE_CSI_VERSION}"
   pullContainerImage "docker" ${CONTAINER_IMAGE}
   echo "  - ${CONTAINER_IMAGE}" >> ${VHD_LOGS_FILEPATH}
 done
 
+CSI_LIVENESSPROBE_WINDOWS_VERSIONS="
+v2.0.1-alpha.1-windows-1809-amd64
```

Comment:
this image is only for windows, what's the right place to put this config @xuto2

Reply:
done. https://github.com/Azure/aks-engine/pull/3448